docs/migration-case.md

```diff
@@ -0,0 +1,44 @@
+# Cluster migration
+
+*Using Backups and Restores*
+
+Heptio Ark can help you port your resources from one cluster to another, as long as you point each Ark instance to the same cloud object storage location. In this scenario, we are also assuming that your clusters are hosted by the same cloud provider. **Note that Heptio Ark does not support the migration of persistent volumes across cloud providers.**
+
+1.  *(Cluster 1)* Assuming you haven't already been checkpointing your data with the Ark `schedule` operation, you need to first back up your entire cluster (replacing `<BACKUP-NAME>` as desired):
+
+    ```
+    ark backup create <BACKUP-NAME>
+    ```
+    The default TTL is 30 days (720 hours); you can use the `--ttl` flag to change this as necessary.
+
+1.  *(Cluster 2)* Make sure that the `BackupStorageLocation` and `VolumeSnapshotLocation` CRDs match the ones from *Cluster 1*, so that your new Ark server instance points to the same bucket.
+
+1.  *(Cluster 2)* Make sure that the Ark Backup object is created. Ark resources are synchronized with the backup files in cloud storage.
```

Comment:
we should note that the default sync interval is 1 minute (as of v0.10), so they can expect a delay of up to that in the backups being imported into cluster 2. And they can adjust that with a server flag if they like